### PR TITLE
ValuesBuilder supports double, Integer, Long, Float, Double types

### PR DESCRIPTION
**Pull request by MyDogTom**

### CODE REVIEW

File: schematic-compiler/src/main/java/net/simonvt/schematic/compiler/TableWriter.java [hunk 1]
```diff
@@ -282,45 +282,30 @@ public void createValuesBuilder(Filer filer, String outPackage) throws IOExcepti
...
-              .returns(ClassName.get(valuesPackage, className))
-              .addParameter(long.class, "value")
-              .addStatement("values.put($T.$L, value)", ClassName.get(columnsClass), column)
-              .addStatement("return this")
-              .build();
-          valuesBuilder.addMethod(longSpec);
+          valuesBuilder.addMethod(
+              makePutMethodSpec(valuesPackage, className, elmName, column, int.class));
+          valuesBuilder.addMethod(
+              makePutMethodSpec(valuesPackage, className, elmName, column, long.class));
```
Comment (SimonVT) [inline schematic-compiler/src/main/java/net/simonvt/schematic/compiler/TableWriter.java]: int and long are redundant, there are only Integer and Long methods on ContentValues.

File: schematic-compiler/src/main/java/net/simonvt/schematic/compiler/TableWriter.java [hunk 2]
```diff
@@ -282,45 +282,30 @@ public void createValuesBuilder(Filer filer, String outPackage) throws IOExcepti
...
-              .returns(ClassName.get(valuesPackage, className))
-              .addParameter(float.class, "value")
-              .addStatement("values.put($T.$L, value)", ClassName.get(columnsClass), column)
-              .addStatement("return this")
-              .build();
-          valuesBuilder.addMethod(floatSpec);
+          valuesBuilder.addMethod(
+              makePutMethodSpec(valuesPackage, className, elmName, column, float.class));
+          valuesBuilder.addMethod(
+              makePutMethodSpec(valuesPackage, className, elmName, column, double.class));
```
Comment (SimonVT) [inline schematic-compiler/src/main/java/net/simonvt/schematic/compiler/TableWriter.java]: Same with double and float.

**Comment by SimonVT** [comment]
Otherwise it looks good :)

**Comment by MyDogTom** [comment]
Removed redundant types.

**Comment by SimonVT** [comment]
Thanks!

**Pull request merged**